2-members.md

```diff
@@ -311,6 +311,28 @@ banner_color: style2
  </section>
 </section>
 
+<!-- 3.5 -->
+<section id="three-five" class="spotlights">
+ <section>
+  <a class="image">
+   <img src="assets/images/members/Dhruv Mendpara.jpeg" alt="" data-position="center center" />
```

Comment:
Have you uploaded the picture?

Reply:
I did upload it and made a pull request for it.